src/main/resources/assets/floodgate/lang/en_US.lang

```diff
@@ -1 +1,4 @@
-tile.floodgate.name=Floodgate
\ No newline at end of file
+tile.floodgate.name=Floodgate
+container.floodgate=Floodgate
+
+gui.block_breaker.enchanted_book.tooltip=Enchanted book goes here
```

Comment:
I assume this is part of the tutorial?

Reply:
Aye, I left it in there so you'd see the tooltip on hover, I'll change it